tests/cypress/e2e/Setup/General/displaypreference_list.cy.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
```

Comment:
```suggestion
```

Reply:
While I agree, I did a copypasta from another cy files, so there is more clean to do ^^